.buildkite/pipeline.yml

```diff
@@ -15,12 +15,12 @@ steps:
    label: crossref-verify
    soft_fail: true
  - commands:
-   - nix-build -A binaries -o binaries
-   - nix-build -A deb-packages -o deb-packages --arg timestamp ${cur_date}
-   - nix-build -A rpm-packages -o rpm-packages --arg timestamp ${cur_date}
+   - "nix-shell -p gnutar --run 'nix-build -A binaries -o binaries'"
```

Comment:
`nix-shell` here fixes the problem with `bzip2` missing, but I think it's better to add `bzip2` to our agent environment, as nix relies on it in run-time

Reply:
This is obviously just a workaround, so yes, ideally we should fix our bk agent.

Reply:
Will you make a PR for our agent config?

Reply:
Interestingly, `gnutar` is already part of `runtimePackages`  for both of our agents.

Reply:
Ah, I think I see the problem, right

Reply:
https://github.com/serokell/serokell-profiles/pull/457

Reply:
Added a PR.